TKeazirian.HTTPServer.Tests/RouterTests.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using TKeazirian.HTTPServer.Tests.helpers;
+using Xunit;
+
+namespace TKeazirian.HTTPServer.Tests;
+
+public class RouterTests
+{
+    private const string NewLine = "\r\n";
+
+    [Fact]
+    public void HandleRequestReturnsEchoBody()
+    {
+        Router router = new Router();
+        var testRequest = HelperFunctions.FormatTestRequest("/echo_body", "POST", "Hello, World");
+
+        string expectedReturn =
```

Comment:
is `Return` a `Response`? How about `expectedRespose` as a name here? `Return` is very generic, and could mean just about anything.